### PR TITLE
Resolve the issues of codestyle of nacos-config module for phase4

### PR DESCRIPTION
**Pull request by wangweizZZ**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

Resolve the issues of codestyle of nacos-config module for some package parts below.
Package:

(1)com.alibaba.nacos.config.server.service.repository

## Brief changelog

1. Remove `@SuppressWarnings("all")` on `PersistService` and fix PMD check 
2. Resolve  other ssues of codestyle of nacos-config module for `repository` part

PersistService is so big ，it contains 109  function, maybe we should split this classes and  remove unused function

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [x] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: config/src/main/java/com/alibaba/nacos/config/server/service/repository/RowMapperManager.java [hunk 1]
```diff
@@ -195,6 +197,7 @@ public ConfigInfoWrapper mapRow(ResultSet rs, int rowNum) throws SQLException {
             try {
                 info.setMd5(rs.getString("md5"));
             } catch (SQLException e) {
+                // ignore
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/service/repository/RowMapperManager.java:200]: change exception name as ignore(d)

File: config/src/main/java/com/alibaba/nacos/config/server/service/repository/RowMapperManager.java [hunk 2]
```diff
@@ -230,6 +233,7 @@ public ConfigInfoBetaWrapper mapRow(ResultSet rs, int rowNum) throws SQLExceptio
...
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/service/repository/RowMapperManager.java:236]: change exception name as ignore(d)

File: config/src/main/java/com/alibaba/nacos/config/server/service/repository/RowMapperManager.java [hunk 3]
```diff
@@ -265,6 +269,7 @@ public ConfigInfoTagWrapper mapRow(ResultSet rs, int rowNum) throws SQLException
...
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/service/repository/RowMapperManager.java:272]: change exception name as ignore(d)

File: config/src/main/java/com/alibaba/nacos/config/server/service/repository/RowMapperManager.java [hunk 4]
```diff
@@ -399,6 +404,7 @@ public ConfigInfo4Beta mapRow(ResultSet rs, int rowNum) throws SQLException {
...
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/service/repository/RowMapperManager.java:407]: change exception name as ignore(d)

File: config/src/main/java/com/alibaba/nacos/config/server/service/repository/RowMapperManager.java [hunk 5]
```diff
@@ -429,6 +435,7 @@ public ConfigInfo4Tag mapRow(ResultSet rs, int rowNum) throws SQLException {
...
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/service/repository/RowMapperManager.java:438]: change exception name as ignore(d)

**Comment by wangweizZZ** [comment]
@KomachiSion Please help to review this pr, thanks.

**Comment by KomachiSion** [comment]
Refer #3249

**Pull request merged**